src/main/java/com/danielflower/apprunner/mgmt/AppManager.java

```diff
@@ -206,7 +206,7 @@ private File fetchChangesAndCreateInstanceDir() throws GitAPIException, IOExcept
         try {
             git.fetch().setRemote("origin").call();
             git.reset().setMode(ResetCommand.ResetType.HARD).setRef("origin/master").call();
-            return copyToNewInstanceDir();
+            return copyToNewInstanceWorkspace();
```

Comment:
Instead of changing the manager and all the runners, can't you make the GoRunner just move the files to a subdir? The calls to get parent dirs in the other runners are confusing now.